lib/merge.js

```diff
@@ -56,6 +68,29 @@ async function merge(context, pullRequest) {
   return true;
 }
 
+async function removeLabels(octokit, pullRequest, mergeRemoveLabels) {
+  const pullRequestLabels = pullRequest.labels.map(label => label.name);
+  const labels = pullRequestLabels.filter(value => mergeRemoveLabels.includes(value))
+
+  if (labels.length < 1) {
+    return;
+  }
+
+  logger.debug("Removing labels...");
+
+  await Promise.all(
+    labels.map(label =>
+      octokit.issues.deleteLabel({
+        owner: pullRequest.head.repo.owner.login,
+        repo: pullRequest.head.repo.name,
+        name: label
+      })
```

Comment:
This needs to include the `issue_number`! See https://octokit.github.io/rest.js/#octokit-routes-issues-remove-label

Also, it needs to be `pullRequest.base` for owner+repo, otherwise it will give problems with forked PRs

Reply:
Fixed.